angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -125,20 +125,9 @@ namespace :db_connector do
 
     sql_owners = "INSERT IGNORE INTO owners
         (name, corporation_name, address_line_one, address_line_two, city, state, zipcode, hpd_registration_id, hpd_registration_contact_id, hpd_type, created_at, updated_at)
-        SELECT CONCAT(TRIM(firstname), ' ', TRIM(middleinitial), ' ', TRIM(lastname)), corporationname, CONCAT(TRIM(businesshousenumber), ' ', TRIM(businessstreetname)),
+        SELECT CONCAT_WS(' ', TRIM(firstname), TRIM(middleinitial), TRIM(lastname)), corporationname, CONCAT(TRIM(businesshousenumber), ' ', TRIM(businessstreetname)),
             businessapartment, businesscity, businessstate, businesszip, registrationid, registrationcontactid, type, NOW(), NOW()
-        FROM #{ENV['MYSQL_DATABASE_DATA']}.hpd_registration_contacts WHERE
-            firstname IS NOT NULL AND
-            firstname != '' AND
-            lastname IS NOT NULL AND
-            lastname != '' AND
-            corporationname IS NOT NULL AND
-            corporationname != '' AND
-            businesshousenumber IS NOT NULL AND
-            businessstreetname IS NOT NULL AND
-            businesscity IS NOT NULL AND
-            businessstate IS NOT NULL AND
-            businesszip IS NOT NULL;"
```

Comment:
Just FYI: I'm guessing the reason this was here was as a first attempt at solving the "duplicate owners" problem. I'll post more in the Slack.